test/modules/aws-security-group-integration.ts

```diff
@@ -234,11 +235,28 @@ describe('Security Group install/uninstall', () => {
     true).then(...finish(done)));
 
   it('uninstalls the Security Group module', uninstall(
-    ['aws_rds', 'aws_ecs_fargate', 'aws_elb', 'aws_security_group', 'aws_ec2', 'aws_vpc'],
+    ['aws_rds', 'aws_ecs_fargate', 'aws_elb', 'aws_security_group', 'aws_ec2'],
   ));
 
+  it('inserts a new VPC (that creates a new default SG automatically)', query(`
+    INSERT INTO vpc (cidr_block)
+    VALUES ('192.${randIPBlock}.0.0/16');
+  `));
+
+  it('creates the VPC', apply());
+
   it('installs the Security Group module', install(
-    ['aws_security_group', 'aws_vpc']));
+    ['aws_security_group']));
+
```

Comment:
should we check the new default SG exists here?

Reply:
Well, I was relying on the install itself not crapping the bed as the test for this, but I can see about how to do that.